.openpublishing.redirection.json

```diff
@@ -333,6 +311,33 @@
         "redirect_document_id": false
       },
 
+      // WebView2 Release Notes:
+      {
+        "source_path": "microsoft-edge/hosting/webview2/releasenotes.md",
+        "redirect_url": "/microsoft-edge/webview2/release-notes/index",
+        "redirect_document_id": false
+      },
+      {
+        "source_path": "microsoft-edge/webview2/releasenotes.md",
+        "redirect_url": "/microsoft-edge/webview2/release-notes/index",
+        "redirect_document_id": false
+      },
+      {
+        "source_path": "microsoft-edge/webview2/release-notes.md",
+        "redirect_url": "/microsoft-edge/webview2/release-notes/index",
+        "redirect_document_id": false
+      },
+      {
+        "source_path": "microsoft-edge/webview2/release-notes/template-prerelease.md",
+        "redirect_url": "/microsoft-edge/webview2/release-notes/index",
+        "redirect_document_id": false
+      },
+      {
```

Comment:
when you try to go to 
https://review.learn.microsoft.com/microsoft-edge/webview2/roadmap/?branch=pr-en-us-3404
where do you end up?
@captainbrosset 

* Moved a group of redirects to better match latest wv2 TOC design.
* Included prefix ./ in new entry, to differentiate the two lines' type of path.  This pattern is globally applied in another PR, makes redirect file easier/ less confusing to maintain.

Reply:
I end up here: https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/release-notes/?branch=pr-en-us-3404&tabs=dotnetcsharp

Reply:
Updated PR main page w/ Ok test result for the redirect.